auth-lib/src/main/java/com/spotify/sdk/android/auth/LoginDialog.java

```diff
@@ -153,16 +155,21 @@ public void onPageStarted(WebView view, String url, Bitmap favicon) {
 
             @Override
             public boolean shouldOverrideUrlLoading(WebView view, String url) {
-                Uri uri = Uri.parse(url);
-                if (url.startsWith(redirectUri)) {
-                    sendComplete(uri);
+                final String caseSafeRequestRedirectUri = redirectUri.toLowerCase(Locale.ENGLISH);
+                final String caseSafeResponseRedirectUri = url.toLowerCase(Locale.ENGLISH);
+                Uri responseUri = Uri.parse(url);
+                if (caseSafeResponseRedirectUri.startsWith(caseSafeRequestRedirectUri)) {
+                    sendComplete(responseUri);
                     return true;
-                } else if (uri.getAuthority().matches(WEBVIEW_URIS)) {
+                } else if (responseUri.getAuthority().matches(WEBVIEW_URIS)) {
                     return false;
                 }
-
-                Intent launchBrowser = new Intent(Intent.ACTION_VIEW, uri);
-                getContext().startActivity(launchBrowser);
```

Comment:
there's a comment above `WEBVIEW_URIS` declaration stating that `When uri that is not whitelisted is received it will be opened in a browser instead.` which is not true anymore.

Reply:
Well spotted